database/seeds/RecordIssuerTypesSeeder.php

```diff
@@ -0,0 +1,25 @@
+<?php
+use Illuminate\Database\Seeder;
+class RecordIssuerTypesSeeder extends Seeder
+// please practice separation of concern, follow laravel way
```

Comment:
Nevermind. We can leave this one as it is. But it's not separation of concern. Technically seeding the `RecordIssuerType` is still `DatabaseSeeder`'s concern